features/upgrade/sdn/multus-upgrade.feature

```diff
@@ -18,63 +17,50 @@
-      | ["items"][0]["metadata"]["name"]                                                           | macvlan-bridge-whereabouts-pod1 |
-      | ["items"][0]["spec"]["template"]["metadata"]["annotations"]["k8s.v1.cni.cncf.io/networks"] | macvlan-bridge-whereabouts      |
-      | ["items"][0]["spec"]["template"]["spec"]["containers"][0]["name"]                          | macvlan-bridge-whereabouts      |
+      | ["items"][0]["spec"]["template"]["metadata"]["labels"]["name"]                             | bridge-static-pod1 |
+      | ["items"][0]["metadata"]["name"]                                                           | bridge-static-pod1 |
+      | ["items"][0]["spec"]["template"]["metadata"]["annotations"]["k8s.v1.cni.cncf.io/networks"] | bridge-static      |
+      | ["items"][0]["spec"]["template"]["spec"]["containers"][0]["name"]                          | bridge-static      |
     Then the step should succeed
     Given a pod becomes ready with labels:
-      | name=test-pod1 |
+    | name=bridge-static-pod1 |
 
-    # Check pod1 has correct macvlan mode on interface net1
-    When I execute on the pod:
-      | ip | -d | link |
-    Then the output should contain:
-      | net1                |
-      | macvlan mode bridge |
     # Check created pod has correct ip address on interface net1
     When I execute on the pod:
       | ip | a |
     Then the output should match:
-      | 192.168.22.10[12] |
+      | 22.2.2.22 |
 
   # @author weliang@redhat.com
   # @case_id OCP-44898
   @admin
   @upgrade-check
-  @users=upuser1,upuser2
   @4.12 @4.11 @4.10 @4.9
-  @vsphere-ipi @openstack-ipi @nutanix-ipi @gcp-ipi @baremetal-ipi @azure-ipi @aws-ipi @alicloud-ipi
-  @vsphere-upi @openstack-upi @nutanix-upi @gcp-upi @baremetal-upi @azure-upi @aws-upi @alicloud-upi
+  @vsphere-ipi @openstack-ipi @gcp-ipi @baremetal-ipi @azure-ipi @aws-ipi
+  @vsphere-upi @openstack-upi @gcp-upi @baremetal-upi @azure-upi @aws-upi
   @upgrade
   @network-ovnkubernetes @network-openshiftsdn
   @proxy @noproxy @disconnected @connected
   Scenario: Check the multus works well after upgrade
     Given I switch to cluster admin pseudo user
     When I use the "multus-upgrade" project
     Given a pod becomes ready with labels:
-      | name=test-pod1 |
-    # Check pod1 has correct macvlan mode on interface net1
-    When I execute on the pod:
-      | ip | -d | link |
-    Then the output should contain:
-      | net1                |
-      | macvlan mode bridge |
+      | name=bridge-static-pod1 |
     # Check created pod has correct ip address on interface net1
     When I execute on the pod:
       | ip | a |
     Then the output should match:
-      | 192.168.22.10[12] |
-
+      | 22.2.2.22 |
     # Delete the created project from testing cluster
-    Given the "multus-upgrade" project is deleted
+    Given the "multus-upgrade" project is deleted
```

Comment:
@weliang1 You may want to add a newline after this line.

Reply:
Looks good except its recommended to not intentionally force delete the namespace in upgrade cases to help improve debugging on shared env.

Reply:
@asood-rh Fixed.

Reply:
> Looks good except its recommended to not intentionally force delete the namespace in upgrade cases to help improve debugging on shared env.

@anuragthehatter Update to keep the project for the debugging.